base/special/trig.jl

```diff
@@ -9,9 +9,11 @@ struct DoubleFloat32
     hi::Float64
 end
 
-# *_kernel functions are only valid for |x| < pi/4 = 0.7854
-# translated from openlibm code: k_sin.c, k_cos.c, k_sinf.c, k_cosf.c
-# which are made available under the following licence:
+# sin_kernel and cos_kernel functions are only valid for |x| < pi/4 = 0.7854
+# translated from openlibm code: k_sin.c, k_cos.c, k_sinf.c, k_cosf.c.
+# acos functions are based on openlibm code: e_acos.c, e_acosf.c.
+# asin functions are based on openlibm code: e_asin.c, e_asinf.c. The above
```

Comment:
I realize this is not true for this commit, but when #23088 is hopefully merged soon it will be.